### ISSUE TITLE
Extension to "commits since" badge: allow counting from latest version tag

### ISSUE THREAD

**Issue by Lucretiel** [badges/shields#1955]
Many projects don't use Github Releases to track releases, but simply ordinary git tags. These show up in the GitHub releases page (for example: https://github.com/Lucretiel/joinery/releases), but aren't marked as releases in the API. It'd be nice if there was another endpoint– perhaps `/commits-since/USER/REPO/latest-tag.svg`– which counts commits after the most recent tag matching `/\d+(\.\d+)/`

**Comment by Lucretiel** [comment]
Prior art: #993

**Comment by paulmelnikow** [comment]
Hi, thanks for the suggestion! It's a nice idea.

To get the latest tag reliably currently requires many API calls (#875), so we may need a caching solution for that first.

**Comment by Lucretiel** [comment]
Yeah, at the end of the day we'd need to implement GitHub paginating. We can fetch up to 100 tags per repo per call, and unfortunately I don't see any obvious way to only fetch new tags.

**Comment by agilgur5** [comment]
So `/github/tag` and `/github/tag-pre` (and `/github/tag-date`) currently get the latest SemVer tag (and date tag). Is it now possible that `commits-since` could extend that logic to allow for commits since latest SemVer?
